lib/commons/aria/index.js

```diff
@@ -1687,7 +1695,15 @@ lookupTable.role = {
 		nameFrom: ['author'],
 		context: null,
 		implicit: ['input[type="search"]'],
-		unsupported: false
+		unsupported: false,
+		allowedElements: [
```

Comment:
General question: How did we miss this, and how do we know we're not missing more stuff this time around?

Reply:
This is the holy grail source of truth - https://www.w3.org/TR/html-aria/ to construct these mappings.

There are 3 caveats with this:
1) Misinterpretation.
2) Conformance between ARIA1.0 and ARIA1.1, that was the case with https://github.com/dequelabs/axe-core/issues/1116. There were multiple ways to use a role. 1.1 mandates usage of role on parent, but backwards compatibility with 1.0 means role can be set on said element itself.
3) When https://www.w3.org/TR/html-aria/ is updated, we need a handle to confirm that things have not been changed, and adapt to new changes. Perhaps can watch/ github hook for the same and streamline the mapping.

To answer your specific Q, we missed this case due to an overlap of 1 and 2 above.

I am going through the table once more to ensure the mappings are right.